### DISCUSSION TITLE
RN fabric & TurboModules

### DISCUSSION THREAD

**Discussion by enahum** [wix/react-native-navigation#7448] [Q&A]
With the upcoming react native release 0.68 the team is introducing Fabric which requires some changes around app initialization and so on.

Currently 0.68 is in rc testing and I was wondering if the wix rnn team is looking into this?

**Comment by SudoPlz** [comment]
Right now react-native 0.68 seems to be incompatible with RNN.

I opened a ticket here: https://github.com/wix/react-native-navigation/issues/7466 to track discussion/what needs to be done to add support.
